packages/react-devtools-extensions/src/astUtils.js

```diff
@@ -40,6 +40,10 @@ function checkNodeLocation(
     return false;
   }
 
+  if (line !== start.line) {
+    return false;
+  }
```

Comment:
At the risk of being picky 😆 the function now has

```js

  if (line < start.line || line > end.line) {
    return false;
  }

  if (line !== start.line) {
    return false;
  }

```

Reply:
hmm yeah good point, let me consolidate that